docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Discover insecure Access Patterns with Teleport Policy
+description: Describes how to enable SSH Keys Scanning using Teleport Policy and Access Graph.
+---
+
+Organizations often struggle to maintain a secure environment due to the lack of visibility into access patterns.
+Long-lived SSH keys, for example, can pose a significant security risk as they can be forgotten or mismanaged while
```

Comment:
```suggestion
Long-lived SSH keys, for example, can pose a significant security risk as they can be compromised or mismanaged while
```

"Forgotten" sounds like a forgotten password, but you don't forget an SSH key in the way you forget a password.